Data/Boards0.json

```diff
@@ -1,2 +1 @@
- ["test",[true,false,18,null,"hello"],3324.34,832432,"world"]
- 
\ No newline at end of file
+ ["test",[true,false,18,null,"hello",[9,8,7,1,2,3]],3324.34,832432,"world"]
```

Comment:
Please add the integer array after `"hello"],` and before `3324.34`. Restrict the size to 2 elements.

The `Board0.json` example is a very minimal string that allows Xilinx Vivado to pass the test in under 1 hour. By extending the test input the time increased exponentially!